UnitsNet.Tests/CustomCode/PressureTests.cs

```diff
@@ -117,5 +120,102 @@ public void PressureDividedByLengthEqualsSpecificWeight()
             SpecificWeight specificWeight = Pressure.FromPascals(20) / Length.FromMeters(2);
             Assert.Equal(SpecificWeight.FromNewtonsPerCubicMeter(10), specificWeight);
         }
+        // Pressure Measurement References
+        [Fact]
+        public void ReferenceConversion_WithDefaultPressureReference_PressureReferenceIsAbsolute()
```

Comment:
May I suggest the following naming scheme to be consistent with existing tests: `WhatIsTested_WhatAreTheInputs_WhatIsExpected`

Following that, I would suggest something like `Reference_WithDefaultPressureReference_IsAbsolute` since we are testing the `Reference` property and we did not pass in an explicit pressure reference in the ctor.

The below method would then be `Gauge_WithDefaultPressureReference_IsOneLessAtmosphere` or something.